src/cli/cli_uart.cpp

```diff
@@ -129,17 +129,17 @@ void Uart::ReceiveTask(const uint8_t *aBuf, uint16_t aBufLength)
     static const char sCommandPrompt[] = {'>', ' '};
 
 #if OPENTHREAD_CONFIG_UART_CLI_RAW
-    if (aBufLength > 0)
+    if (aBufLength > 0 && mRxLength + aBufLength < kRxBufferSize)
```

Comment:
When `OPENTHREAD_CONFIG_UART_CLI_RAW=1`, are we guaranteed that `aBuf` does not contain multiple commands (e.g. two sub-strings ending in `\r\n` that are concatenated)?

Reply:
Good point. I will enhance this.